README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
This is an unexpected issue, could it be because the nodes on the client side aren't getting namespaced properly?  
Are you starting the client and server simultaneously?